src/community/script/py/src/test/resources/org/geoserver/script/py/factorial.py

```diff
@@ -1,2 +1,3 @@
+from functools import reduce
```

Comment:
This is for the geoscript plugin which uses Jython, no idea if this will still work in Jython :)

Reply:
See the discussion at jython/book#6#